libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -183,8 +183,15 @@ export function getExperienceName() {
 
 export function loadStyles(path) {
```

Comment:
Should you use the `loadStyles` method instead for the [unav?](https://github.com/adobecom/milo/pull/2402/files#diff-628bf5a25903978c5b36da4943ec813da171346d24bd64f80907f012a840b098R520)

Otherwise you duplicate the error handling behavior

Reply:
Done.
Using `loadStyles` instead of `loadStyle` to add log for the static assets like css.